apps/src/templates/progress/progressHelpers.js

```diff
@@ -286,6 +286,7 @@ export const levelProgressFromServer = serverProgress => {
     result: getLevelResult(serverProgress),
     paired: serverProgress.paired || false,
     timeSpent: serverProgress.time_spent || 0,
+    lastTimestamp: serverProgress.last_progress_at || 0,
```

Comment:
Is there a reason this needs to be `0` by default, why not just have it be undefined?

Reply:
i made the `lastTimestamp` prop required in `studentLevelProgressType`. my thinking is that it _should_ be there so it should be required, but that parsing should fail gracefully if it's missing. we should discuss and standardize when to use `required` in prop types when we have our north star discussion about typing. in the meantime, if you think it makes more sense to have this be an optional field i'm totally open to that (but i'd love to hear your reasoning).

Reply:
Ok, I understand. I was thinking about what the value represents, for example, what if we wanted to sort on timestamp in the future, having a number would mean it has a value and is sortable vs if we used undefined or null which explicitly represent a missing value. But this makes more sense if it's never supposed to be missing